examples/comp-demo/Embed.toml

```diff
@@ -0,0 +1,46 @@
+[probe]
```

Comment:
You might want to update to cargo-embed v0.8 and change these to [default.probe] etc.